packages/ui-kit/src/components/TimeRangePicker/TimeRangePicker.const.ts

```diff
@@ -0,0 +1,78 @@
+import { DateRangeOptionsProps } from './TimeRangePicker.types'
+import { RelativeTimeRange } from '../../helpers'
+
+export const defaultOptions: DateRangeOptionsProps[] = [
+  {
+    uid: 'today',
+    label: 'Today',
+    value: {
+      relative: RelativeTimeRange.Today
+    }
+  },
+  {
+    uid: 'this-week',
+    label: 'This week',
+    value: {
+      relative: RelativeTimeRange.ThisWeek
+    }
+  },
+  {
+    uid: 'this-month',
+    label: 'This month',
+    value: {
+      relative: RelativeTimeRange.ThisMonth
+    }
+  },
+  {
+    uid: 'this-year',
+    label: 'This year',
+    value: {
+      relative: RelativeTimeRange.ThisYear
+    }
+  },
+  {
+    uid: 'last-7-days',
+    label: 'Last 7 days',
+    value: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 7
+    }
+  },
+  {
+    uid: 'last-30-days',
+    label: 'Last 30 days',
+    value: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 30
+    }
+  },
+  {
+    uid: 'last-90-days',
+    label: 'Last 90 days',
+    value: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 90
+    }
+  },
+  {
+    uid: 'last-365-days',
+    label: 'Last 365 days',
+    value: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 365
+    }
+  }
+]
+
+export const lastNOptions = [
+  { label: 'days', value: RelativeTimeRange.LastNDays },
+  { label: 'weeks', value: RelativeTimeRange.LastNWeeks },
+  { label: 'months', value: RelativeTimeRange.LastNMonths },
+  { label: 'years', value: RelativeTimeRange.LastNYears }
+]
+
+export const DATE_FORMAT = 'MM/dd/yyyy'
+export const CUSTOM_RANGE_FORMAT = 'MMMM dd, yyyy, HH:mm:ss'
```

Comment:
My concern with these hard-coded formats is, how will they localize? I tend to think we should bend our designs a little to just pass through whatever the browser thinks is the best locale-aware time representation. Then, we don't have to go figure out how it should be displayed for every user's locale.

```ts
new Date().toLocaleString('en-US', { dateStyle: 'long', timeStyle: 'medium' })
// April 26, 2024 at 4:36:35 PM

new Date().toLocaleString('de-DE', { dateStyle: 'long', timeStyle: 'medium' })
// 26. April 2024 um 16:37:50

new Date().toLocaleString('en-DE', { dateStyle: 'long', timeStyle: 'medium' })
// 26 April 2024 at 16:38:12
```

Reply:
That was a really good catch, thanks! I've added the locale support.

https://github.com/propeldata/ui-kit/assets/55801864/6ac3d5ac-c3be-481b-badc-325f41250bba